database/events/common_schemata.py

```diff
@@ -66,6 +66,47 @@
 }
 
 
+SOMMERAKADEMIE = {
+    'type': 'object',
+    'properties': {
+        'phone': {
+            'type': 'string',
+            'enum': ['Physik', 'Medizinphysik', 'Lehramt'],
```

Comment:
That doesn't look right ;)

Reply:
Whoops, its right in my local db entry now :D